### ISSUE TITLE
Cannot seem to use yolov1 GPU to train

### ISSUE THREAD

**Issue by ystan9** [pjreddie/darknet#360]
I am trying to train using tiny yolov1, but not able to get any result even though CPU only works. It is as follows for using GPU:
layer     filters    size              input                output
    0 conv     64  7 x 7 / 2   448 x 448 x   3   ->   224 x 224 x  64
    1 max          2 x 2 / 2   224 x 224 x  64   ->   112 x 112 x  64
    2 conv    192  3 x 3 / 1   112 x 112 x  64   ->   112 x 112 x 192
    3 max          2 x 2 / 2   112 x 112 x 192   ->    56 x  56 x 192
    4 conv    128  1 x 1 / 1    56 x  56 x 192   ->    56 x  56 x 128
    5 conv    256  3 x 3 / 1    56 x  56 x 128   ->    56 x  56 x 256
    6 conv    256  1 x 1 / 1    56 x  56 x 256   ->    56 x  56 x 256
    7 conv    512  3 x 3 / 1    56 x  56 x 256   ->    56 x  56 x 512
    8 max          2 x 2 / 2    56 x  56 x 512   ->    28 x  28 x 512
    9 conv    256  1 x 1 / 1    28 x  28 x 512   ->    28 x  28 x 256
   10 conv    512  3 x 3 / 1    28 x  28 x 256   ->    28 x  28 x 512
   11 conv    256  1 x 1 / 1    28 x  28 x 512   ->    28 x  28 x 256
   12 conv    512  3 x 3 / 1    28 x  28 x 256   ->    28 x  28 x 512
   13 conv    256  1 x 1 / 1    28 x  28 x 512   ->    28 x  28 x 256
   14 conv    512  3 x 3 / 1    28 x  28 x 256   ->    28 x  28 x 512
   15 conv    256  1 x 1 / 1    28 x  28 x 512   ->    28 x  28 x 256
   16 conv    512  3 x 3 / 1    28 x  28 x 256   ->    28 x  28 x 512
   17 conv    512  1 x 1 / 1    28 x  28 x 512   ->    28 x  28 x 512
   18 conv   1024  3 x 3 / 1    28 x  28 x 512   ->    28 x  28 x1024
   19 max          2 x 2 / 2    28 x  28 x1024   ->    14 x  14 x1024
   20 conv    512  1 x 1 / 1    14 x  14 x1024   ->    14 x  14 x 512
   21 conv   1024  3 x 3 / 1    14 x  14 x 512   ->    14 x  14 x1024
   22 conv    512  1 x 1 / 1    14 x  14 x1024   ->    14 x  14 x 512
   23 conv   1024  3 x 3 / 1    14 x  14 x 512   ->    14 x  14 x1024
   24 conv   1024  3 x 3 / 1    14 x  14 x1024   ->    14 x  14 x1024
   25 conv   1024  3 x 3 / 2    14 x  14 x1024   ->     7 x   7 x1024
   26 conv   1024  3 x 3 / 1     7 x   7 x1024   ->     7 x   7 x1024
   27 conv   1024  3 x 3 / 1     7 x   7 x1024   ->     7 x   7 x1024
   28 Local Layer: 7 x 7 x 1024 image, 256 filters -> 7 x 7 x 256 image
   29 dropout       p = 0.50               12544  ->  12544
   30 connected                            12544  ->   784
   31 Detection Layer
forced: Using default '0'
Loading weights from darknet.conv.weights...Done!
Learning Rate: 0.0005, Momentum: 0.9, Decay: 0.0005
Loaded: 1.446220 seconds
Detection Avg IOU: 0.000000, Pos Cat: 0.000000, All Cat: 0.000000, Pos Obj: 0.000000, Any Obj: 0.000000, count: 18
Detection Avg IOU: 0.000000, Pos Cat: 0.000000, All Cat: 0.000000, Pos Obj: 0.000000, Any Obj: 0.000000, count: 19
Detection Avg IOU: 0.000000, Pos Cat: 0.000000, All Cat: 0.000000, Pos Obj: 0.000000, Any Obj: 0.000000, count: 19
Detection Avg IOU: 0.000000, Pos Cat: 0.000000, All Cat: 0.000000, Pos Obj: 0.000000, Any Obj: 0.000000, count: 18
1: 27.096424, 27.096424 avg, 0.000500 rate, 9.151883 seconds, 64 images
Loaded: 0.000042 seconds

why does this happen?I am using GTX 1060 and added arch for sm_61

**Comment by lizhiyuanUSTC** [comment]
I also have these problems, I am trying to find a older version darknet.

**Comment by CBIR-LL** [comment]
Download the previous commit.

**Comment by ystan9** [comment]
How long ago commit should I be looking at

**Comment by ystan9** [comment]
Problem solve, I revert back to Nov 22, 2016 version of darknet and it is possible to train

**Comment by lizhiyuanUSTC** [comment]
Hi, @ystan9 , there is an error about CUDNN when compiling the older version. Which version of CUDNN do you use when compiling the older darknet? I am using CUDA8.0 and CUDNN v6.

**Comment by ystan9** [comment]
I did not use CUDNN, I enabled only GPU = 1
